docker-to-ecr.sh

```diff
@@ -3,9 +3,7 @@
 set -e
 
 # get aws account id if it does not exist in env var
-AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID:-$(aws sts get-caller-identity --query "Account")}
-AWS_ACCOUNT_ID="${AWS_ACCOUNT_ID%\"}"
-AWS_ACCOUNT_ID="${AWS_ACCOUNT_ID#\"}"
+AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID:-$(aws sts get-caller-identity --query "Account" --output text)}
```

Comment:
I'm concerned about outputting the account id here, if that's what this is doing.